packages/smooth_app/lib/pages/scan/continuous_scan_page.dart

```diff
@@ -96,20 +81,44 @@ class ContinuousScanPage extends StatelessWidget {
                         )
                       ],
                     ),
-                    Column(
-                      mainAxisAlignment: MainAxisAlignment.spaceBetween,
-                      children: <Widget>[
-                        Container(
-                          padding: const EdgeInsets.only(top: 32.0),
-                          child: Column(
+                    if (_continuousScanModel.isNotEmpty)
+                      Column(
+                        mainAxisAlignment: MainAxisAlignment.end,
+                        children: <Widget>[
+                          Row(
+                            mainAxisAlignment: MainAxisAlignment.spaceEvenly,
                             children: <Widget>[
-                              ScanPage.getContributeChooseToggle(
-                                _continuousScanModel,
-                                context,
+                              ElevatedButton.icon(
+                                icon: const Icon(CupertinoIcons.clear_circled),
+                                onPressed: () async =>
+                                    _continuousScanModel.clearScanSession(),
+                                label: const Text('clear'),
```

Comment:
```suggestion
                                label: const Text('Clear'),
```

Reply:
Could you add all the new strings in appLocalizations, so that we can translate them?

Reply:
I prefer not to: I have no idea how writing localizations works in that project.

Anyway, that would be safe to play before with the UI, for instance to see if two large buttons each in the full screen width are better than two smaller contiguous buttons, in order to give appropriate guidelines to the translators.